DirectEmployers/MyJobs
# [PD-3249] date range

## What's this PR do?
Changes the Date Range Selection in Analytics to not go back any farther than 5/4/2017

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside any of the analytics applications

## How should this be manually tested?
Pull down the repository. Go into Analytics applications and select a date from the date range calendar. You will see that it automatically wont' let you go back any farther than May 4th and if one of the quick selections is selected it still goes from May 4th as the start

## Any background context you want to provide?
Most of this is hard coded right now and needs to be for DEAM until we figure out how to make it dynamic
